index.html

```diff
@@ -188,21 +206,36 @@ <h2 class="heading">About</h2>
             </ul>
             <img class="icon-bg" src="images/ic_skills.svg" alt="#" />
           </li>
+          <li class="about heading-pic">
+            <img
+              class="icon-bg"
+              src="images/image_dots_with_teal.svg"
+              alt="#"
+            />
+          </li>
         </ul>
       </section>
-      <section class="form-container">
+      <section class="form-container" id="contact">
         <form
```

Comment:
- [ ] **Required:** Excellent implementation of the form, however, you have not included the focus color on the border. Let's add it to signify the selected/active input element.

Your Implementation | Figma Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/149516480-495c8736-b00b-4fec-8b9a-12c41e94a91c.png) | ![image](https://user-images.githubusercontent.com/27709832/149516503-9c2f4fc6-89e2-4e01-8e4f-ae9ca9b080c1.png)